src/components/tuCollapse/tuCollapseItem.vue

```diff
@@ -14,19 +10,17 @@
 				<tu-icon :icon-pack="iconPack" :icon="iconArrow" />
 			</span>
 		</header>
-		
-		<div
-			ref="content"
-			:style="styleContent"
-			class="tu-collapse-item--content"
-		>
-			<hr style="border-top: 1px solid #ffffff55; margin-bottom: 5px"/>
-
-			<div class="con-content--item">
-				<slot />
+		</summary>
+			<div
+				ref="content"
+			>
```

Comment:
If there are no multiple params, you can just compress the div onto a single line `<div ref="content">`. Saves lines and number of changes